gui/src/controls/image.ts

```diff
@@ -159,21 +159,34 @@ module BABYLON.GUI {
             return this._cellWidth;
         }
         set cellWidth(value: number) {
+            if(this._cellWidth === value){
+                return;
+            }
+            
             this._cellWidth = value;

```

Comment:
missing the _markAsDirty here

Reply:
Didn't add it as it's usually not something changed on-the-go, but i suppose it's possible someone has different sized images in the same texture.

Reply:
added.